_zh-cn/overviews/scala3-book/introduction.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 导言
+type: chapter
+description: This page begins the overview documentation of the Scala 3 language.
```

Comment:
Do you want to also translate this one?

Reply:
Type and description are invisible to readers, so we'd better leave them as they are.

Reply:
Clarification: type is related to the rendering of the page, and description is invisible to readers.